src/libraries/Microsoft.PowerFx.Core/Utils/GetTokensUtils.cs

```diff
@@ -35,7 +35,11 @@ internal static class GetTokensUtils
                         case BindKind.Control:
                             tokens[item.Name] = TokenResultType.HostSymbol;
                             break;
+                        case BindKind.OptionSet:
```

Comment:
Nit, why not combine Control, OptionSet and PowerFxResolved object into a single case block